resources/views/index.blade.php

```diff
@@ -689,6 +689,11 @@ class="my-prism-editor"
                     // remove \n from string that is used for display
                     var url = doc.url.replace(/\n/g, '')
 
+                    const isLoginUrl = url.split('/').pop() === '{{config('request-docs.last_login_path_name')}}';
+                    const isLogoutUrl = url.split('/').pop() === '{{config('request-docs.last_logout_path_name')}}';
+                    const tokenResponseName = '{{config('request-docs.token_response_name')}}';
```

Comment:
I think just the token name would do and can be attached to any path. There is no need to specify specific paths.

Reply:
I just added to become easy to use like postman because postman has this feature you can set environment from response to avoid every time copy and paste the token when login or remove token when logout

Reply:
We can just set the header as global and in general, there won't be a need to specify `isLoginUrl` etc.

Reply:
but must specify the login url to get the token from its response and set it to header

Reply:
Hmm I don’t understand still. 
why specify a route, just set the header whenever there is a response and it has the token.

Reply:
Ok you now i understand you are right we can remove it only check the token in all responses